frontend/src/auth/components/PrivateRoute/PrivateRoute.tsx

```diff
@@ -19,11 +24,11 @@ export const PrivateRoute: React.FC<RouteProps> = (props: RouteProps) => {
 
   if (!isAuthenticated) {
     const renderComponent = () => (
-      <Redirect to={{ pathname: ROUTES.Auth.LOGOUT }} />
+      <Redirect to={{ pathname: fallbackRedirect }} />
     );
     return (
       <Route path={props.path} component={renderComponent} render={undefined} />
     );
   }
-  return <Route path={props.path} component={props.component} />;
+  return <Route {...props} />;
```

Comment:
With this change you can now provide a fallback path for PrivateRoutes that is different from the regular logout route.